src/routes/user/user.routes.js

```diff
@@ -13,19 +14,19 @@ const userRoutes = (router) => {
     return res.status(StatusCodes.CREATED).json(response);
   });
 
-  router.post('/login', async (req, res) => {
+  router.get('/login', async (req, res) => {
     const response = await userLogin(req.body);
 
+    res.cookie('auth', response.message.token, { maxAge: 900000, httpOnly: true });
```

Comment:
```suggestion
    res.cookie('auth', response.token, { maxAge: 900000, httpOnly: true });
```

maxAge should be in `.env`